.github/workflows/haskell.yml

```diff
@@ -8,12 +8,12 @@ jobs:
     runs-on: ubuntu-16.04
     strategy:
       matrix:
-        ghc: ["8.2.2", "8.4.4", "8.6.5", "8.8.1"]
+        ghc: ["8.2.2", "8.4.4", "8.6.5", "8.8.2"]
         cabal: ["3.0"]
 
     steps:
     - uses: actions/checkout@v2
-      if: github.event.action == 'opened' || github.event.action == 'synchronize'
+      if: github.event.action == 'opened' || github.event.action == 'synchronize' || github.event.ref == 'refs/heads/master'
```

Comment:
We also want to run on pushes to `master` (tho we should try to avoid making those in general).